### DISCUSSION TITLE
Blade pitch control

### DISCUSSION THREAD

**Discussion by ghost** [NatLabRockies/ROSCO#92] [Q&A] [answered]
Hello.

I have a quick question. I am using OrcaFlex+Wind turbine module and ROSCO is utilized for controller. 
I use shutdown mode. My problem is that once shutdown mode is operated, it does not go back to operational mode.
Sometimes, due to ramping time, even before 25 m/s, shutdown occurs. Once shutdown occurs, it does not go back to operational mode even if the wind velocity is below 25m/s.  

Is it normal?

**Comment by nikhar-abbas** [comment, marked as answer]
Hi @jinch999,
This is the expected behavior of ROSCO - once it is shutdown, it will be for the remainder of the simulation. 

Nikhar

**Reply by ghylander** [reply]
AFAIK, OrcaFlex does not allow for initial conditions, ie you cannot start a simulation with the turbine in a near-operation state. This could mean that if using shutdown mode, the controller will enter shutdown mode before the turbine actually begins operation.

Can't think of a direct workaround to this

@jinch999, you could try using the OrcaFlex python wrapper to setup a script that waits a certain amount of time before calling the actual ROSCO controller, that way you would avoid the controller detecting shutdown conditions while the simulation ramps up

**Reply by nikhar-abbas** [reply]
Sorry about the delay here - 
Another option is to add some sort of "wait" in the ROSCO source code. This would involve something like putting [this line](https://github.com/NREL/ROSCO/blob/9646dbdab604471477c40c33023ec776a9ac9ec0/ROSCO/src/ControllerBlocks.f90#L395) in an `if` statement like:
```
IF (LocalVar%Time > 60.0) THEN
    LocalVar%SD  = .TRUE.
END IF
```

The wait time could be added as an input in the DISCON.IN file if there is enough reason to do so.
